services/apps/src/apps/clients/metadata.clj

```diff
@@ -0,0 +1,66 @@
+(ns apps.clients.metadata
+  (:use [kameleon.uuids :only [uuidify]])
+  (:require [cemerick.url :as curl]
+            [cheshire.core :as json]
+            [clj-http.client :as http]
+            [apps.util.config :as config]
+            [apps.util.service :as util]))
+
+(defn- metadata-url-encoded
+  [& components]
+  (str (apply curl/url (config/metadata-base) (map curl/url-encode components))))
+
+(defn- get-options
+  [params]
+  {:query-params     params
+   :as               :stream
+   :follow-redirects false})
+
+(defn- post-options
+  [body params]
+  {:query-params     params
+   :body             body
+   :content-type     :json
+   :as               :stream
```

Comment:
Just a small convenience: if you use `:as :json` instead then `clj-http` will parse the JSON and store the resulting data structure in the `:body` key.